beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/service/PagingCleanupService.java

```diff
@@ -88,6 +90,15 @@ private void cleanupContent(EntityHousekeepingPath housekeepingPath) {
       log.info("Cleaning up path \"{}\"", housekeepingPath.getPath());
       pathCleaner.cleanupPath(housekeepingPath.getPath(), housekeepingPath.getTableName());
       updateAttemptsAndStatus(housekeepingPath, PathStatus.DELETED);
+    } catch (MultiObjectDeleteException e) {
```

Comment:
I don't think this is the place to do it. I think it should be done in the S3PathCleaner. That's the class that knows about AWS and this is an AWS exception that should bubble up. If we change the implementation of PathCleaner this catch will be useless.